geonode_mapstore_client/client/js/plugins/ResourcesGrid.jsx

```diff
@@ -429,6 +431,9 @@ function ResourcesGrid({
     ],
     pagePath = '',
     pageSize = 24,
+    panel,
+    cardLayoutStyle = null,
+    defaultLayoutCardsStyle = 'grid',
```

Comment:
We should review the names because if we are using cardLayoutStyle the default should have the same structure eg:

- **cardLayoutStyle** 
- default**CardLayoutStyle** 

I'm open to changes but the two prop name should be consistent